Library/Formula/wireshark.rb

```diff
@@ -3,13 +3,9 @@ class Wireshark < Formula
   homepage "https://www.wireshark.org"
 
   stable do
-    url "https://www.wireshark.org/download/src/all-versions/wireshark-1.12.8.tar.bz2"
-    mirror "https://1.eu.dl.wireshark.org/src/wireshark-1.12.8.tar.bz2"
-    sha256 "357e0a4e49525d80cdc740bb16539fcdb526ad38cc2ed6cabedafc9bdee5c7e7"
-
-    # Removes SDK checks that prevent the build from working on CLT-only systems
-    # Reported upstream: https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=9290
-    patch :DATA
+    url "https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0.tar.bz2"
```

Comment:
I could build 2.0.0 successfully on fresh Mavericks after ```xcode-select --install```.
There is something different from CI environment probably. How can I build the same env as Jenkins?

```zsh
$ brew install wireshark
==> Downloading https://homebrew.bintray.com/bottles/wireshark-2.0.0.mavericks.bottle.tar.gz

curl: (22) The requested URL returned error: 404 Not Found
Error: Failed to download resource "wireshark"
Download failed: https://homebrew.bintray.com/bottles/wireshark-2.0.0.mavericks.bottle.tar.gz
Warning: Bottle installation failed: building from source.
==> Installing dependencies for wireshark: pkg-config
==> Installing wireshark dependency: pkg-config
==> Downloading https://homebrew.bintray.com/bottles/pkg-config-0.29.mavericks.bottle.tar.gz
######################################################################## 100.0%
==> Pouring pkg-config-0.29.mavericks.bottle.tar.gz
🍺  /usr/local/Cellar/pkg-config/0.29: 10 files, 652K
==> Downloading https://www.wireshark.org/download/src/all-versions/wireshark-2.0.0.tar.bz2
######################################################################## 100.0%
==> ./configure --disable-silent-rules --prefix=/usr/local/Cellar/wireshark/2.0.0 --with-gnutls --disable-wireshark --disable-gtktest --
==> make
==> make install
==> Caveats
If your list of available capture interfaces is empty
(default OS X behavior), try the following commands:

  curl https://bugs.wireshark.org/bugzilla/attachment.cgi?id=3373 -o ChmodBPF.tar.gz
  tar zxvf ChmodBPF.tar.gz
  open ChmodBPF/Install\ ChmodBPF.app

This adds a launch daemon that changes the permissions of your BPF
devices so that all users in the 'admin' group - all users with
'Allow user to administer this computer' turned on - have both read
and write access to those devices.

See bug report:
  https://bugs.wireshark.org/bugzilla/show_bug.cgi?id=3760
==> Summary
🍺  /usr/local/Cellar/wireshark/2.0.0: 315 files, 66M, built in 9.1 minutes
$ brew test wireshark
Testing wireshark
==> Using the sandbox
==> /usr/local/Cellar/wireshark/2.0.0/bin/randpkt -b 100 -c 2 capture.pcap
==> /usr/local/Cellar/wireshark/2.0.0/bin/capinfos -Tmc capture.pcap
```

```zsh
$ brew config
HOMEBREW_VERSION: 0.9.5
ORIGIN: (none)
HEAD: (none)
Last commit: never
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: dual-core 64-bit arrandale
OS X: 10.9.5-x86_64
Xcode: N/A
CLT: 6.2.0.0.1.1424975374
Clang: 6.0 build 600
X11: N/A
System Ruby: 2.0.0-p481
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /usr/bin/ruby => /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/bin/ruby
Java: N/A
```

```zsh
$ otool -L /usr/local/bin/tshark
/usr/local/bin/tshark:
        /usr/local/Cellar/wireshark/2.0.0/lib/libwiretap.5.dylib (compatibility version 6.0.0, current version 6.0.0)
        /usr/local/Cellar/wireshark/2.0.0/lib/libwireshark.6.dylib (compatibility version 7.0.0, current version 7.0.0)
        /usr/local/opt/geoip/lib/libGeoIP.1.dylib (compatibility version 8.0.0, current version 8.7.0)
        /usr/local/Cellar/wireshark/2.0.0/lib/libwsutil.6.dylib (compatibility version 7.0.0, current version 7.0.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1197.1.1)
        /usr/local/opt/glib/lib/libgthread-2.0.0.dylib (compatibility version 4601.0.0, current version 4601.2.0)
        /usr/local/opt/glib/lib/libgmodule-2.0.0.dylib (compatibility version 4601.0.0, current version 4601.2.0)
        /usr/local/opt/glib/lib/libglib-2.0.0.dylib (compatibility version 4601.0.0, current version 4601.2.0)
        /usr/local/opt/gettext/lib/libintl.8.dylib (compatibility version 10.0.0, current version 10.4.0)
        /usr/lib/libpcap.A.dylib (compatibility version 1.0.0, current version 1.0.0)
        /usr/local/opt/c-ares/lib/libcares.2.dylib (compatibility version 4.0.0, current version 4.0.0)
        /System/Library/Frameworks/Kerberos.framework/Versions/A/Kerberos (compatibility version 5.0.0, current version 6.0.0)
        /System/Library/Frameworks/SystemConfiguration.framework/Versions/A/SystemConfiguration (compatibility version 1.0.0, current version 596.15.0)
        /usr/local/opt/libgcrypt/lib/libgcrypt.20.dylib (compatibility version 21.0.0, current version 21.4.0)
        /usr/local/opt/libgpg-error/lib/libgpg-error.0.dylib (compatibility version 17.0.0, current version 17.0.0)
        /usr/local/opt/gnutls/lib/libgnutls.28.dylib (compatibility version 70.0.0, current version 70.10.0)
        /usr/lib/libz.1.dylib (compatibility version 1.0.0, current version 1.2.5)
        /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 855.17.0)
```